### DISCUSSION TITLE
CocOpenFloat equivalent event for pum?

### DISCUSSION THREAD

**Discussion by creasty** [neoclide/coc.nvim#4016] [Q&A] [answered]
Would like to modify `&winhighlight` of the custom popup menu.

https://github.com/creasty/dotfiles/blob/58d9edbdc5b0286695c2e5a281f70931a12d7b4a/nvim/autoload/user/plugin/coc.vim#L79-L82

**Comment by chemzqm** [comment, marked as answer]
`CocOpenFloat` is fired for the pum, and seems you don't need to remove `Search` highlight since it's not added to `winhighlight`.

**Reply by creasty** [reply]
No, it actually removes Search highlight **by adding** `Search:` to `&winhighlight`, and it works well with normal floats but not with pum.

### Normal floats

With `&winhighlight=Search:`:

<img width="1074" alt="Screen Shot 2022-08-08 at 8 44 32" src="https://user-images.githubusercontent.com/1695538/183315766-0e9de2a5-d1ec-4b1a-a051-236a1af72e1b.png">

Without `&winhighlight=Search:`:

<img width="1120" alt="Screen Shot 2022-08-08 at 8 47 41" src="https://user-images.githubusercontent.com/1695538/183315863-04e80200-70c0-4b25-a0ed-e5a30720538a.png">

### PUM

Highlight remains:

<img width="636" alt="Screen Shot 2022-08-08 at 8 43 56" src="https://user-images.githubusercontent.com/1695538/183315869-b483ef25-8541-48f0-9ff0-c47fef1dc2f0.png">
